examples/CmdServer.hs

```diff
@@ -50,17 +50,16 @@ time sk = Clock.getTime Clock.Monotonic >>= sendValue sk
 random :: Socket -> IO ()
 random sk = (randomIO :: IO Int) >>= sendValue sk
 
-def :: (String, Socket) -> IO ()
-def (str, sk) = sendValue sk ("Unknown command: " ++ str)
+def :: Socket -> IO ()
+def sk = sendValue sk "Unknown command"
 
-commands :: Map.Map String (Fold IO Socket ())
-commands = Map.fromList
-    [ ("time"  , Fold.drainBy time)
-    , ("random", Fold.drainBy random)
-    ]
+keytoFold :: String -> IO (Fold IO Socket ())
+keytoFold "time" = return (Fold.drainBy time)
+keytoFold "random" = return (Fold.drainBy random)
+keytoFold _ = return (Fold.drainBy def)
```

Comment:
1. Keep the original name "commands"
2. Use a case statement instead of writing separate equations for each case.